File: DigitalSolutions.TestApp.WebAPI/DigitalSolutions.TestApp.WebAPI/DTO/FilterDTO.cs
using DigitalSolutions.TestApp.WebAPI.DataBase.DBContext;

namespace DigitalSolutions.TestApp.WebAPI.DTO
{
    public class FilterDTO
    {
        public string? minCreateTime { get; set; }
        public string? maxCreateTime { get; set; }
        public string? minChangeTime { get; set; }
        public string? maxChangeTime { get; set; }
        public int minAccountNumber { get; set; }
        public int maxAccountNumber { get; set; }
        public string? accountStatus { get; set; }
        public long minBalance { get; set; }
        public long maxBalance { get; set; }
        public string? paymentMethod { get; set; }
    }
}

File: DigitalSolutions-TestApp-UI/src/app/services/FilterDTO.ts
import { Injectable } from "@angular/core";

@Injectable()
export class FilterDTO
{
    minCreateTime : string = "2000-01-01T00:00:00.0000000";
    maxCreateTime : string = "2100-01-01T00:00:00.0000000";
    minChangeTime : string = "2000-01-01T00:00:00.0000000";
    maxChangeTime : string = "2100-01-01T00:00:00.0000000";
    minAccountNumber : number = 0;
    maxAccountNumber : number = 9999;
    accountStatus : string = "0";
    minBalance : number = 0;
    maxBalance : number = 9999;
    paymentMethod : string = "0";
}